### DISCUSSION TITLE
How to upgrade?

### DISCUSSION THREAD

**Discussion by pixeluvw** [appwrite/appwrite#3247] [Q&A] [answered]
hi,
How to upgrade my digtal ocean Appwrite image without breaking things? Any help would be welcome

**Comment by maeddin** [comment, marked as answer]
Just go to your Droplet and upgrade as described in the [docs](https://appwrite.io/docs/upgrade). Of course, make sure you have a backup before you do this.

**Reply by stnguyen90** [reply]
An additional resource: Video on [Staying Up to Date with Appwrite Upgrades](https://youtu.be/YKFfVk_nqVY).

An additional emphasis: All non-patch upgrades require a migration (e.g. 1.0.3 to 1.1.0 requires a migration).

Lastly, on DigitalOcean, your appwrite folder should be in the the root's home directory.

**Comment by tripolskypetr** [comment]
If you facing the 500 Internal Server Error while using AppWrite Cloud data migration, [that json-per-document migration tool](https://github.com/react-declarative/appwrite-backup-tool) will be quite useful
